backend/src/main/java/com/google/sps/notifs/EmailNotifier.java

```diff
@@ -128,32 +67,47 @@ private static Message createMessageWithEmail(MimeMessage emailContent)
    * @return the MimeMessage to be used to send email
    * @throws MessagingException if there was a problem accessing the Store
    */
-  private MimeMessage createEmail(String subject, String bodyText) throws MessagingException {
+  private MimeMessage createEmail(String recipientUsername, String subject, String bodyText)
+      throws MessagingException {
 
     Properties props = new Properties();
     Session session = Session.getDefaultInstance(props, /* authenticator= */ null);
 
     MimeMessage email = new MimeMessage(session);
 
     email.setFrom(new InternetAddress(APPLICATION_EMAIL));
-    email.addRecipient(RecipientType.TO, new InternetAddress(recipientEmail));
+    email.addRecipient(RecipientType.TO, new InternetAddress(recipientUsername + "@google.com"));
```

Comment:
is there any way we can avoid baking the @google.com assumption into this class? For instance could we get the email address from the authenticated user in the servlet and then pass the full email address to this method?

Reply:
Removed the @google.com prefix !